repo/packages/C/confluent-kafka/10/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "packagingVersion": "3.0",
+  "name": "confluent-kafka",
+  "version": "1.1.16-3.1.1",
+  "minDcosReleaseVersion": "1.7",
+  "maintainer": "partner-support@confluent.io",
+  "description": "Apache Kafka by Confluent",
```

Comment:
Added link to Confluent on DC/OS white paper and Community support group.   Will update to the DC/OS examples doc when that is complete.